csharp/autobuilder/Semmle.Autobuild.Shared/Autobuilder.cs

```diff
@@ -310,6 +322,18 @@ void exitCallback(int ret, string msg, bool silent)
                 .Select(result => result.ToDiagnosticMessage(this, diagSeverity))
                 .ForEach(AddDiagnostic);
 
+            if (buildResult == 0 && IsBuildless)
```

Comment:
Should we add a `"buildless/failed"` diagnostic if we get here with `buildResult != 0`?

Reply:
If it helps the buildless telemetry dashboard, we can add it too. I think there's no such diagnostic for java at the moment.

Reply:
I've added the `csharp/autobuilder/buildless/failed`.